label_studio/organizations/api.py

```diff
@@ -103,7 +104,17 @@ def get_serializer_context(self):
 
     def get_queryset(self):
         org = generics.get_object_or_404(self.request.user.organizations, pk=self.kwargs[self.lookup_field])
-        return org.members.order_by('user__username')
+        if flag_set('fix-backend-dev-3134-exclude-deactivated-users', self.request.user):
+            serializer = OrganizationsParamsSerializer(data=self.request.GET)
+            serializer.is_valid(raise_exception=True)
+            active = serializer.validated_data.get('active')
+            if active or not bool_from_request(self.request.GET, 'contributed_to_projects', False):
```

Comment:
why can't we use serializer to get `contributed_to_projects` param?

Reply:
Fixed